report_print.rb

```diff
@@ -62,6 +62,40 @@ def print_report_summary(report)
   puts
 end
 
+def print_report_motd(report, motd_path)
+  motd = []
+  header = "# #{report.host} #"
+  headline = "#" * header.size
+  motd << headline << header << headline << ''
+
+  motd << "Last puppet run happened at %s in environment %s." % [report.time, report.environment]
+
+  motd << "The result of this puppet run was "
+  case report.status
+  when 'failed'
+    motd.last << color(:red, report.status)
```

Comment:
I do not know how good Ruby it as deleting tempfiles in case of errors (e.g syntax error?)